client/packages/system/src/IndicatorsDemographics/DetailView/PercentageColumn.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { BasicTextInput } from '@common/components';
+import { ColumnDefinition, RecordWithId } from '@openmsupply-client/common';
+
+interface RecordWithIdWithPercentageFields extends RecordWithId {
+  percentage?: number | null;
+  name?: string;
+}
+
+export const percentageColumn = <
+  T extends RecordWithIdWithPercentageFields,
+>(): ColumnDefinition<T> => {
+  return {
+    label: 'label.percentage',
+    setter: () => {
+      if (process.env['NODE_ENV']) {
+        throw new Error(
+          `The default setter of the NameAndColor column was called.
+        Have you forgotten to provide a custom setter?
+        When setting up your columns, you should provide a setter function
+        const columns = useColumns([ gercentageColumn(), { setter }])
+        `
+        );
+      }
+    },
+    accessor: ({ rowData }) => rowData.percentage,
+    key: 'percentage',
+    Cell: ({ rowData, column, isDisabled }) => (
+      <Box
+        sx={{
+          flexDirection: 'row',
+          borderBottom: 'none',
+          alignItems: 'center',
+          display: 'flex',
+        }}
+      >
+        <>
+          <BasicTextInput
+            disabled={isDisabled || rowData.name == 'General Population'}
```

Comment:
Using rowData.id === `1` would be safer. the 'gen pop' will need translation-ability

Reply:
Oh, and using `==` doesn't test for strict equality. best to use `===` if you can